File: 3264-final-array-state-after-k-multiplication-operations-i/3264-final-array-state-after-k-multiplication-operations-i.cpp
class Solution {
public:
    vector<int> getFinalState(vector<int>& nums, int k, int multiplier) {
        while(k--)
        {
            int min = 0;
            for(int i=1; i<nums.size(); i++)
            {
                if(nums[i]<nums[min])
                    min = i;
            }
            nums[min] *= multiplier;
        }
        return nums;
    }
};

File: 3264-final-array-state-after-k-multiplication-operations-i/3264-final-array-state-after-k-multiplication-operations-i.py
class Solution:
    def getFinalState(self, nums: List[int], k: int, multiplier: int) -> List[int]:
        for _ in range(k):
            mini=min(nums)
            for i,num in enumerate(nums):
                if(num==mini):
                    nums[i]=num*multiplier
                    break
        return nums